### ISSUE TITLE
Accessibility: Link missing a text alternative

### ISSUE THREAD

**Issue by sarah-cul** [cul-it/uls#1265]
### Description
because the social media icons at the bottom of the unit sites have aria-hidden="true" and they don't have an alternative text, the placement of the screen reader span is inside the icon element prevents the screen readers from actually reading it

### Suggested solution

Move the sr-only span outside of the icon element so screen readers can access it
```

<a href="https://www.youtube.com/@cornelllibrary">
    <i class="fa-brands fa-youtube" aria-hidden="true"></i>
    <span class="sr-only">YouTube</span>
</a>

```

**Comment by sarah-cul** [comment]
fixed https://github.com/cul-it/wp-cul-theme-culu/releases/tag/v2.3.2
